s3_file_field/_multipart.py

```diff
@@ -1,9 +1,7 @@
-from __future__ import annotations
```

Comment:
@dchiquito I just noticed, why is this getting removed? This is what allows the `MultipartManager` type annotation to be forward-referenced without making it a string.

Reply:
I missed that I had stringly typed `MultipartManager` and thought it was unused. I've restored it.